src/bin/cql-stress-cassandra-stress/settings/param/parser.rs

```diff
@@ -147,6 +148,21 @@ impl ParamsParser {
         MultiParamHandle::new(param)
     }
 
+    pub fn distribution_param(
+        &mut self,
+        prefix: &'static str,
+        default: Option<&'static str>,
+        desc: &'static str,
+        required: bool,
+    ) -> DistributionParamHandle {
+        let param = Rc::new(RefCell::new(DistributionParam::new_wrapped(
+            prefix, default, desc, required,
+        )));
+
+        self.params.push(Rc::clone(&param) as ParamCell);
+        DistributionParamHandle::new(param)
+    }
```

Comment:
Why can't we use an existing `simple_param`/`simple_subparam`/`multi_param` for the distribution?

Reply:
`simple_param` parsing logic could be reused for sure. The only difference is help-printing logic.
Look at the part of output of `help -insert`:
```
$ ./cassandra-stress help -insert

Usage: -insert [revisit=DIST(?)] [visits=DIST(?)] partitions=DIST(?) [batchtype=?] select-ratio=DIST(?) row-population-ratio=DIST(?)

  [revisit=DIST(?)]: The distribution with which we revisit partial writes (see visits); implicitly defines size of revisit collection
      EXP(min..max)                        An exponential distribution over the range [min..max]
      EXTREME(min..max,shape)              An extreme value (Weibull) distribution over the range [min..max]
      QEXTREME(min..max,shape,quantas)     An extreme value, split into quantas, within which the chance of selection is uniform
      GAUSSIAN(min..max,stdvrng)           A gaussian/normal distribution, where mean=(min+max)/2, and stdev is (mean-min)/stdvrng
      GAUSSIAN(min..max,mean,stdev)        A gaussian/normal distribution, with explicitly defined mean and stdev
      UNIFORM(min..max)                    A uniform distribution over the range [min, max]
      FIXED(val)                           A fixed distribution, always returning the same value
      SEQ(min..max)                        A fixed sequence, returning values in the range min to max sequentially (starting based on seed), wrapping if necessary.
      Preceding the name with ~ will invert the distribution, e.g. ~exp(1..10) will yield 10 most, instead of least, often
      Aliases: extr, qextr, gauss, normal, norm, weibull

... rest of parameters ...
```

In addition to parameters info (name, usage etc.), the distribution parameters list all of the available distributions.

Reply:
Would it make sense to adjust `simple_param` so that it optionally supports printing additional help?

Reply:
Yes, I think so.